build.gradle

```diff
@@ -91,7 +91,7 @@ include {
 	}
 
 	from('modules/shared/logging.gradle') {
-		slf4jAndLogback '1.7.10', '1.0.13'
+		slf4jAndLogback '1.7.25', '1.0.13'
```

Comment:
Did changing logback to `1.2.12` cause any problems?

Reply:
Why should I change it to a lower version?

Reply:
The first version is the SLF4J version, with you changes at `1.7.25`, the second version is the logback version, now at `1.0.13`.